src/Microsoft.ML.AutoML/AutoMLExperiment/AutoMLExperiment.cs

```diff
@@ -261,14 +261,19 @@ public async Task<TrialResult> RunAsync(CancellationToken ct = default)
                         monitor.ReportBestTrial(trialResult);
                     }
                 }
-                catch (Exception)
+                catch (Exception ex)
                 {
                     if (cts.Token.IsCancellationRequested)
                     {
                         break;
                     }
                     else
                     {
+                        // TODO
```

Comment:
I agree. I wouldn't abort the whole thing for a single failure (though there still should be some threshold. Not a blocker for this PR, just agreeing with your comment.